packages/MainUI/.env.production

```diff
@@ -1,3 +1,5 @@
 NEXT_PUBLIC_API_BASE_URL="http://localhost:8080/etendo"
 NEXT_PUBLIC_CACHE_DURATION="3600000"
 NEXT_PUBLIC_AUTH_HEADER_NAME="Authorization"
+NEXT_PUBLIC_BASIC_AUTH_USERNAME=production_user
+NEXT_PUBLIC_BASIC_AUTH_PASSWORD=secure_password
```

Comment:
This is going to be an issue, since this constants are bunlded in the client side bundle, so, an attacker could read the credentials from the source code